### DISCUSSION TITLE
Want to contribute Polish translation

### DISCUSSION THREAD

**Discussion by TurtleIdiot** [astubenbord/paperless-mobile#82] [General]
Hello, I'm a native Polish speaker and would like to provide a translation for Polish to the app!

**Comment by astubenbord** [comment]
Hi, thanks! Just send me an email I can use to invite you to localizely

**Reply by TurtleIdiot** [reply]
makskolis@gmail.com

**Reply by astubenbord** [reply]
Did you get an invitation? :)

**Reply by TurtleIdiot** [reply]
Yes, thank you! I'll get to it later today as I have been busy this week
